test/distributed/test_symmetric_memory.py

```diff
@@ -1124,10 +1124,9 @@ def func_3(x):
 
 
 class SymmMemSingleProcTest(TestCase):
-    @skipIfRocm
     @requires_cuda
     @skipIf(
-        _get_torch_cuda_version() < (12, 0),
+        not TEST_WITH_ROCM and _get_torch_cuda_version() < (12, 0),
```

Comment:
RoCM does not have a version requirement?

Reply:
No, not a cuda/rocm version requirement. There might be gpu driver requirement. I will think some more on how to incorporate that into the code.